frontend/src/modules/EditZing/Components/GroupCard.tsx

```diff
@@ -122,17 +125,27 @@ export const GroupGrid = ({
             }}
           />
         </Box>
-        <Grid container spacing={2}>
+        <Box
+          sx={{
+            display: 'grid',
+            gridTemplateColumns: 'repeat(auto-fit, minmax(112px, max-content))',
+            gap: '16px',
```

Comment:
I think `gap: 2` may also work here if the default gap size is 8px (uses theme spacing)